charts/csi-driver-lvm/README.md

```diff
@@ -1,5 +1,9 @@
 # CSI Driver LVM Helm Chart
 
+This is the helm chart for the deployment of https://github.com/metal-stack/csi-driver-lvm.
+
+**The chart requires that you have at least Kubernetes 1.19.**
```

Comment:
```suggestion
**The chart requires that you have at least Kubernetes 1.16.**
```

Reply:
Is this really the case? I read this page here: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#storage-resources-v122 and it says "CSIDriver is available in storage.k8s.io/v1 since v1.19."? Or is this resource not required?

Reply:
https://github.com/metal-stack/helm-charts/blob/d47a1c0110c5f8699bcd88d496673d4a394f74e0/charts/csi-driver-lvm/templates/common.yaml#L1-L6

Reply:
Raised this to 1.19 because the docs say so.

Reply:
Was able to test it with minikube now, so it appears to work:

```
❯ k get node 
NAME       STATUS   ROLES    AGE   VERSION
minikube   Ready    master   27s   v1.16.15
❯ helm upgrade --install csi-driver-lvm ./charts/csi-driver-lvm --values tests/csi-driver-lvm/values.yaml --wait
Release "csi-driver-lvm" does not exist. Installing it now.
NAME: csi-driver-lvm
LAST DEPLOYED: Mon Sep 19 10:27:32 2022
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
❯ k get po -A 
NAMESPACE     NAME                               READY   STATUS    RESTARTS   AGE
default       csi-driver-lvm-controller-0        3/3     Running   0          25s
default       csi-driver-lvm-plugin-jnm66        3/3     Running   0          25s
kube-system   coredns-5644d7b6d9-6v8zk           1/1     Running   0          2m33s
kube-system   etcd-minikube                      1/1     Running   0          103s
kube-system   kube-apiserver-minikube            1/1     Running   0          95s
kube-system   kube-controller-manager-minikube   1/1     Running   0          109s
kube-system   kube-proxy-vkcxp                   1/1     Running   0          2m33s
kube-system   kube-scheduler-minikube            1/1     Running   0          94s
kube-system   storage-provisioner                1/1     Running   1          2m47s